src/interfaces/AdHocMeasureTitle.ts

```diff
@@ -0,0 +1,24 @@
+// (C) 2007-2018 GoodData Corporation
+/**
+ * The interface of the object used by the {ArithmeticMeasureTitleFactory} to not be dependent on the
+ * {VisualizationObject.IMeasure}. It contains all the necessary properties related to the measure title.
+ *
+ * @internal
+ */
+export interface IMeasureTitleProps {
+    localIdentifier: string;
+    title?: string;
+    alias?: string;
+}
+
+/**
+ * The interface of the object used by the {ArithmeticMeasureTitleFactory} to not be dependent on the
+ * {VisualizationObject.IArithmeticMeasureDefinition}. It contains all the necessary properties from the arithmetic
```

Comment:
This comment explains some historical reasoning which I understand but it would be sufficient to explain just the current state - something around "properties necessary to generate arithmetic measure name"

Reply:
Fixed.